### DISCUSSION TITLE
awake_msg method was deleted

### DISCUSSION THREAD

**Discussion by lbk861** [fltk-rs/fltk-rs#1515] [Q&A]
Hello, I found awake_msg was deleted in the latest version. Also I noticed the [issue](https://github.com/fltk-rs/fltk-rs/issues/483). And it' why? Unavoidable memory leaks?

**Comment by MoAlyousef** [comment]
It’s just hidden:
https://github.com/fltk-rs/fltk-rs/blob/3c9b7a1dde7a3869a09d4ca74a1abd304386fa24/fltk/src/app/channel.rs#L21
